src/database.ts

```diff
@@ -1336,7 +1336,7 @@ class Database extends GrpcServiceObject {
         reqOpts,
         gaxOpts,
       },
-      (err, sessions, ...args) => {
+      (err, sessions, nexPageRequest, ...args) => {
```

Comment:
`nextPageRequest`

Reply:
Thanks,
Fixed here and everywhere below.